android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java

```diff
@@ -1167,4 +1170,20 @@ public Map<String, Object> getConstants() {
 
         return constants;
     }
+
+    @ReactMethod
+    public void setOnFeaturesUpdatedListener() {
+        InternalCore.INSTANCE._setOnFeaturesUpdatedListener(new OnFeaturesUpdatedListener() {
```

Comment:
what's the different between _setOnFeaturesUpdatedListener and _setFeaturesStateListener ?

Reply:
Honestly, I'm not familiar with `_setFeaturesStateListener`; Android provides `_setOnFeaturesUpdatedListener` specifically for network flag updates.

Reply:
Andrew Safwat from the Android team confirmed that the `_setOnFeaturesUpdatedListener` will be the new listener the team will maintain in the future. This change is because the `_setFeaturesStateListener` lacks extensibility; for instance, APM cannot utilize it.